packages/website/docs/SingleSelect.md

```diff
@@ -1,24 +1,49 @@
-## Guidelines
+# Best Practices
 
-### General
+Use single-select to allow user to select one option from a **long list that can be filtered**. Single-selects are more appropriate when the available list of options is very long or when the space is limited.
 
--   If space allows, and there are no more than 7 options to choose from, consider using [radio buttons](#/components/Radio) instead.
--   Keep option labels as short as possible in order to limit width to a minimum.
+Use alphanumerical order unless a more suited ordering rational applies (such as sizes or security levels).
 
-### Options Sorting
+Always include the ability to filter the list when there are over 20 options.
 
--   Always put the most common items at the top of the list. If no option is more common than the others, use alphanumeric or chronological order.
+## Labeling
 
-### Label
+Keep titles and labels short, **under one line**.
 
--   In most cases, you should include a label to identify the component.
+### Titles
```

Comment:
Should the `title` (best practice) section be under Forms?

Reply:
We'll probably have a more general guidelines relating to labeling and copy writing. Is that what you're saying?

Reply:
We'll probably have a more general guidelines relating to labeling and copy writing. Is that what you're saying?

Reply:
If I understand correctly, wouldn't titles be part of a form section?  I see titles, then labels.

Reply:
The title here refers to the name given to the single-select input. 
Let's say you have to choose an ice cream flavor, from a predefined list. 

Then there would be a title "Flavor" above the single-select where all flavor options would be. 
![image](https://user-images.githubusercontent.com/44175746/152019449-67f9bfd7-8597-4808-8f3f-71558f29d586.png)